src/content/docs/mobile-monitoring/new-relic-mobile-ios/ios-sdk-api/set-max-event-pool-size.mdx

```diff
@@ -28,10 +28,12 @@ Compatible with all agent versions.
 
 ## Description
 
-By default, the iOS agent collects a maximum of 1000 events per [harvest cycle](/docs/accounts-partnerships/education/getting-started-new-relic/glossary#harvest-cycle). (Cycles are ten minutes long by default). When the pool size reaches its limit, the agent will begin [sampling events](/docs/agents/manage-apm-agents/agent-data/new-relic-events-limits-sampling), discarding some old and some new events, until the pool of events are transmitted with the next harvest cycle.
-
+By default, the iOS agent collects a maximum of 1000 events per [harvest cycle](/docs/accounts-partnerships/education/getting-started-new-relic/glossary#harvest-cycle). 
+This method controls the maximum size of the event pool stored in the memory until the next harvest cycle. When the pool size limit is reached, the New Relic Android agent will begin [sampling events](/docs/agents/manage-apm-agents/agent-data/new-relic-events-limits-sampling), discarding some old and some new events, until the pool of events are transmitted with the next harvest cycle. 
 This method lets you override the maximum size of that event pool. You must set this value after the start call.
 
+* The default value for the event harvest cycle is 600 seconds. 
```

Comment:
Same as for Android: seems it should refer to the maxPoolSize() method instead?

Reply:
see above